### PR TITLE
restructure xaiengine install rules to match sysroot structure

### PR DESCRIPTION
**Pull request by denolf**

### CODE REVIEW

File: tutorials/CMakeLists.txt [hunk 1]
```diff
@@ -52,6 +52,8 @@ set(MLIR_BINARY_DIR ${CMAKE_BINARY_DIR})
 find_package(Vitis)
 find_package(Python3 COMPONENTS Interpreter)
 
+message(STATUS "NOLF NOLF TUTORIALS LibXAIE Stuff")
```
Comment (fifield) [inline tutorials/CMakeLists.txt:55]: I'm not sure this message is useful :)

**Review by fifield — APPROVED** [review]
Make sense although to be honest I comment out the some of the xaiengine build rules because it doesn't work right for me.

**Comment by denolf** [comment]
merged #417

File: runtime_lib/xaiengine/CMakeLists.txt [hunk 2]
```diff
@@ -13,6 +13,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 project("xaiengine driver")
 set(XAIE_SOURCE ${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_2/src)
 file(GLOB libsources ${XAIE_SOURCE}/*/*.c ${XAIE_SOURCE}/*/*/*.c)
+file(GLOB libheaders ${VITIS_AIETOOLS_DIR}/include/drivers/aiengine/*.h)
```
Comment (stephenneuendorffer) [inline runtime_lib/xaiengine/CMakeLists.txt:16]: At least in 2022.2, this path doesn't seem to point anywhere?

**Pull request merged**